### PR TITLE
Automatically create and push policy bundles

### PR DESCRIPTION
**Pull request by simonbaird**

### CODE REVIEW

**Comment by simonbaird** [comment]
The GitHub action seems to be working, see https://github.com/simonbaird/ec-policies/actions/runs/3316419981/jobs/5478191610 .

**Comment by simonbaird** [comment]
I added my Azure push password at the org level, see https://github.com/organizations/hacbs-contract/settings/secrets/actions .

**Review by zregvart — APPROVED** [review]

File: .github/workflows/push-bundles.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Docker login
+      uses: docker/login-action@v2
+      with:
+        # See also BUNDLE_REPO defined in Makefile which should match
+        registry: quay.io/hacbs-contract
+        username: ${{ secrets.EC_POLICY_BUNDLE_PUSH_USERNAME }}
+        password: ${{ secrets.EC_POLICY_BUNDLE_PUSH_PASSWORD }}
```
Comment (simonbaird) [inline .github/workflows/push-bundles.yaml:23]: Do the secret names feel too long? Could make them QUAY_PUSH_USER and QUAY_PUSH_PASS probably..

File: Makefile [hunk 2]
```diff
@@ -236,6 +237,58 @@ check-pipeline-opa: ## Run policy evaluation for pipeline using opa. Deprecated.
...
+
+# Pushes two bundles, one for release and one for policy.
+# Each bundle includes policy/lib and its contents,
+# which is why we need the temp dir and the extra copying.
+# $(*) is expected to be either "release" or "pipeline".
+#
+.PHONY: push-bundles
+
+BUNDLE_REPO=quay.io/hacbs-contract
+BUNDLE_PREFIX=$(BUNDLE_REPO)/ec-policy
```
Comment (simonbaird) [inline Makefile:250]: I was considering just /policy here to make the repo name shorter, e.g. `https://quay.io/repository/hacbs-contract/ec-policy-release` vs `https://quay.io/repository/hacbs-contract/policy-release`.

Could also consider `https://quay.io/repository/hacbs-contract/release-policy` maybe for extra readability.

**Review by joejstuart — APPROVED** [review]

**Pull request merged**